src_docs/source/test_results/nightly_system_tests.rst

```diff
@@ -0,0 +1,4 @@
+Nightly System Tests Pipelines
+==============
```

Comment:
title underlines should be at least as long as the title

Reply:
ok; thnak you